refactor: use cool-seq-tool feature_overlap method for matching cnvs

austinant opened the pull request:
*used feature_overlap endpoint
*rewrote auxiliary functions/matching code to fit with feature_overlap data structure *removed gene overlap code
*replaced/updated Jaccard score

korikuzma commented:
I'm running this using the python api. I'll make a PR into this once it's finished running.

korikuzma commented:
@austinant You have these comments for nch and clinvar:
`remember to replace df1 with df to actually loop over full dataset`
`remember to replace dv1 with dv to actually loop over full dataset`

However, I cannot find `df1` or `dv1` doing a CMD+F

austinant commented:
I fixed up the code and removed all the df1s and dv1s, this is now done in
one cell early in the nb that replaces df and dv with their restrictions to
the Y chromosome. I just missed clearing out the comments, it looks like

On Wed, Nov 15, 2023 at 13:04 Kori Kuzma ***@***.***> wrote:

> @austinant <https://github.com/austinant> You have these comments for nch
> and clinvar:
> remember to replace df1 with df to actually loop over full dataset
> remember to replace dv1 with dv to actually loop over full dataset
>
> However, I cannot find df1 or dv1 doing a CMD+F
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/GenomicMedLab/variation-normalizer-manuscript/pull/76#issuecomment-1813099723>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AF7HKEEB6UYKX6ETPRAQXPLYEUG2LAVCNFSM6AAAAAA7MZZTFSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTQMJTGA4TSNZSGM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>

korikuzma commented:
> I fixed up the code and removed all the df1s and dv1s, this is now done in one cell early in the nb that replaces df and dv with their restrictions to the Y chromosome. I just missed clearing out the comments, it looks like
> […](#)
> On Wed, Nov 15, 2023 at 13:04 Kori Kuzma ***@***.***> wrote: @austinant <https://github.com/austinant> You have these comments for nch and clinvar: remember to replace df1 with df to actually loop over full dataset remember to replace dv1 with dv to actually loop over full dataset However, I cannot find df1 or dv1 doing a CMD+F — Reply to this email directly, view it on GitHub <[#76 (comment)](https://github.com/GenomicMedLab/variation-normalizer-manuscript/pull/76#issuecomment-1813099723)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AF7HKEEB6UYKX6ETPRAQXPLYEUG2LAVCNFSM6AAAAAA7MZZTFSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTQMJTGA4TSNZSGM> . You are receiving this because you were mentioned.Message ID: ***@***.*** com>

Okay cool. Just wanted to double check! Thanks

korikuzma commented:
I'm also doing some cleanup in my branch

korikuzma commented:
@austinant we'll hold off merging this until my changes get merged in. Going to create a PR once it's done re-running.
